metrics/web/lib/common/domain/usecases/fetch_instant_config_usecase.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:async';
+
+import 'package:metrics/base/domain/usecases/usecase.dart';
+import 'package:metrics/common/domain/entities/instant_config.dart';
+import 'package:metrics/common/domain/repositories/instant_config_repository.dart';
+import 'package:metrics/common/domain/usecases/parameters/instant_config_param.dart';
+
+/// A [UseCase] that provides an ability to fetch the [InstantConfig].
+class FetchInstantConfigUseCase
+    extends UseCase<FutureOr<InstantConfig>, InstantConfigParam> {
+  /// An [InstantConfigRepository] that provides an ability to interact
+  /// with the persistent store.
+  final InstantConfigRepository _repository;
+
+  /// Creates the [FetchInstantConfigUseCase] use case with
+  /// the given [InstantConfigRepository].
+  ///
+  /// Throws an [ArgumentError] if the [InstantConfigRepository] is `null`.
+  FetchInstantConfigUseCase(this._repository) {
+    ArgumentError.checkNotNull(_repository, 'repository');
+  }
+
+  @override
+  FutureOr<InstantConfig> call(InstantConfigParam params) async {
+    try {
+      return await _repository.fetch();
```

Comment:
Let's handle possible `null` values here so we can set default via code for testing. For example, if we introduce a new config `testConfig` we can set its default value and test the application without setting this value in the database.